demo/apisix/conf/apisix.yaml

```diff
@@ -86,9 +86,37 @@ services:
         bearer_only: true
         introspection_endpoint_auth_method: "client_secret_post"
         redirect_uri: "http://test.sso.example.com:8080/login-callback.html"
+      api-breaker:
+        break_response_code: 503
+        max_breaker_sec: 300 # should be var: JMAP_CIRCUIT_BREAKER_TIMEOUT
+        unhealthy:
+          http_statuses:
+            - 500
+            - 501
+            - 502
+            - 503
+            - 504
+          failures: 3  # should be var: JMAP_CIRCUIT_BREAKER_MAXERRORS
+        healthy:
+          successes: 1
+
   -
     id: jmap_no_auth_service
     upstream_id: jmap_upstream
+    plugins:
+      api-breaker:
```

Comment:
The double declaration is redundant. I believe we could manage to declare the plugin only once. Look at my PR. Declaring a plugin and using `plugin_config_id`, even if other plugins are declared seem to work (which I like, avoid some useless verbosity like in krakend)